src/GeositeFramework/plugins/launchpad/style.css

```diff
@@ -26,3 +26,41 @@
     list-style-type: none;
     padding: 0;
 }
+
+.truncate-toggle {
+    padding: 2px !important;
+    font-size: 0.7em !important;
+    width: 35%;
+    float: none !important;
```

Comment:
In this case, you could increase the specificity of the rule to `.launchpad-plugin .plugins .button.truncate-toggle` to avoid using `!important`.